lstgen/generators/base.py

```diff
@@ -139,9 +146,9 @@ def _write_else(self, stmt):
 
     def _convert_exec(self, expr):
         (var, parsed_stmt) = parse_eval_stmt(expr)
-        ret = ['this.', var, ' = ']
+        ret = [self.instance_var+'.', var, ' = ']
```

Comment:
ret = [self.instance_var, '.', var, ' = ']

Reply:
don't use `this` for go with `self.instance_var` see abdove.

Reply:
Ok, will rename